lib/eventFactory.js

```diff
@@ -18,17 +18,24 @@ let instance;
  * @return {Array}                                          Array of commit jobs to start
```

Comment:
We should need to add `config.branch` to JSDoc.

Reply:
Thanks I will add that to JSDoc.